stock_picking_info_lot/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,4 @@
+This feature allows for inventory operations to have additional information about the lot numbers being moved.
```

Comment:
@ZavarinArtem  Could you a little bit be more accurate about 'additional information' is ?

Reply:
Thank you for the review.
I've rewritten the description, can you look is it enough information now?